e2e/cephfs.go

```diff
@@ -528,6 +528,23 @@ var _ = Describe("cephfs", func() {
 				}
 			})
 
+			By("Create PVC, bind it to an app, unmount volume and check app deletion", func() {
+				pvc, app, err := createPVCAndAppBinding(pvcPath, appPath, f, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC and application with error %v", err)
```

Comment:
Use wrapped static errors instead of dynamic. You can refer this PR: https://github.com/humblec/ceph-csi/commit/82b2ad28a6e3b578af701c32bd112e761e9ec602